### PR TITLE
UICIRC-648: Add RTL/Jest tests for `GeneralSection` component

### PR DESCRIPTION
**Pull request by Stanislau-Snapok**
## Purpose
Add RTL/Jest testing for `GeneralSection` component in `RequestPolicy/components/EditSections`

## Refs
https://issues.folio.org/browse/UICIRC-648

## Screenshots
image

### CODE REVIEW

File: src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ class GeneralSection extends React.Component {
 
     return (
       <Accordion
+        data-testid="generalRequestPolicyFormTestId"
```
Comment (pavelspichonak) [inline src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.js:62]: as discussed should remove `TestId` postfix

File: src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.test.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      const expectedResult = {
+        id: 'request_policy_name',
+        name: 'name',
+        required: true,
+        autoFocus: true,
+        component: TextField,
+        validate: mockedValidateName,
+      };
+
+      expect(Field).toHaveBeenNthCalledWith(1, expect.objectContaining(expectedResult), {});
```
Comment (pavelspichonak) [inline src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.test.js:90]: as discussed indexes => map

**Review by annamelnyk — APPROVED** [review]

File: src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.js (hunk 1 above)
Comment (Stanislau-Snapok) [reply src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.js]: done

File: src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.test.js (hunk 2 above)
Comment (Stanislau-Snapok) [reply src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.test.js]: done

File: src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.js (hunk 1 above)
Comment (Stanislau-Snapok) [reply src/settings/RequestPolicy/components/EditSections/GeneralSection/GeneralSection.js]: done

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Pull request merged**